sdcm/nemesis.py

```diff
@@ -3332,6 +3332,14 @@ def _run_commands_wait_and_cleanup(  # pylint: disable=too-many-arguments
                 self.log.debug(f"{name}: failed to execute cleanup command "
                                f"{cmd} on node {node} due to the following error: {str(exc)}")
 
+    def wait_restarted_node_fully_start(self, node):
```

Comment:
looks like this is following https://github.com/scylladb/scylla-cluster-tests/pull/2663 footsteps.

and seems like a pattern across multiple places (and not just those too), we should consider moving this function into cluster or node

Reply:
we can consider it in a PR where we can organize everything.. as the purpose of this PR is to fix a persistent issue we are seeing while trying to release 5.2